### PR TITLE
Add Datagrip as client

### PR DESCRIPTION
**Pull request by ryandoowan**

### CODE REVIEW

File: _data/tools.yml [hunk 1]
```diff
@@ -625,6 +625,21 @@
...
+  anchor: jetbrains-datagriop
+  category: client
+  logo: /assets/images/logos/datagrip.png
+  logosmall: /assets/images/logos/datagrip-small.png
+  description: |
+    DataGrip by JetBrains is an IDE for databases that is tailored to suit the
+    specific needs of professional SQL developers. It is designed to work with
+    databases installed locally, on a server, or in the cloud. It is installed
+    as a local application on your workstation.
+  links:
```
Comment (mosabua) [inline _data/tools.yml:638]: Check with the others like DBeaver if we say that it uses the JDBC driver

File: _data/tools.yml [hunk 2]
```diff
@@ -625,6 +625,21 @@
...
+    - urltext: JetBrains Datagrip
+      url: https://www.jetbrains.com/datagrip/
+    - urltext: Datagrip Documentation
```
Comment (mosabua) [inline _data/tools.yml:641]: this is not the datagrip documentation .. its from starburst.. so maybe just call it a tutorial

Datagrip with Trino tutorial

**Review by mosabua — CHANGES_REQUESTED** [review]

File: _data/tools.yml (hunk 1 above)
Comment (ryandoowan) [reply _data/tools.yml:638]: we did not

**Review by mosabua — APPROVED** [review]
Looks good now.

**Pull request merged**